docs/shape_batch_size_and_layout.md

```diff
@@ -29,3 +29,12 @@ it ignores the batch_size value.
 on [Shape Inference Document](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_ShapeInference.html).
 In case the model can't be reshaped, it will remain in the original parameters and all requests with incompatible input format
 will get an error. The model server will also report such problem in the logs.
+
+# Changing model input/output layout
+- Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can specify layout during conversion to IR format via Model Optimizer.
+- Only 2 layouts are supported: `NCHW` and `NHWC`.
```

Comment:
Maybe it would be a good idea to extend those abbreviations and explain to the reader what letters stand for + one example of how it determines shape like:

N - elements in the batch
C - color channels
H - image height
W - image width
Example for resnet input:
For NCHW layout the input shape is: (1, 3, 224, 224)
For NHWC layout the input shape is (1, 224, 224 ,3)

Reply:
done